diff --git a/node_modules/aki-api/src/lib/functions/Request.js b/node_modules/aki-api/src/lib/functions/Request.js
index b2446ad..9a2c1dd 100644
--- a/node_modules/aki-api/src/lib/functions/Request.js
+++ b/node_modules/aki-api/src/lib/functions/Request.js
@@ -21,10 +21,12 @@ const params = {
 
 const rp = async (uri) => {
   try {
+    console.log(uri)
     const result = await axios.get(uri, {
       headers,
       params,
     });
+    console.log(result);
     const beginningParse = result.data.indexOf('(');
     const jsonString = result.data.substring(beginningParse + 1, result.data.length - 1);
     result.data = JSON.parse(jsonString);
